composer.json

```diff
@@ -1,8 +1,8 @@
 {
     "name": "mention-me/emarsys",
-    "description": "Emarsys RestFull API client",
+    "description": "Emarsys API client",
     "require": {
-        "php": ">=7.3",
+        "php": "^8.2",
```

Comment:
Given tests are passing, and we're running this in a PHP 8.2 env (the platform) I think we're fine.